src/sonic-config-engine/minigraph.py

```diff
@@ -1174,6 +1180,13 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         }
     }
 
+    if kube_data:
+        results['KUBERNETES_MASTER'] = { 'SERVER': {
+            'disable': 'True' if kube_data.get('disable', '0') == '0' else 'False',
```

Comment:
>'di [](start = 12, length = 3)

Indent inner block one more level?

Reply:
This is the default indent provided by vi editor for .py script for a dict object.

Reply:
I checked with a python formatter.
```python
    if kube_data:
        results["KUBERNETES_MASTER"] = {
            "SERVER": {
                "disable": "True" if kube_data.get("disable", "0") == "0" else "False",
                "disable": str(kube_data.get("enable", "0") == "0"),
                "ip": kube_data.get("ip", ""),
            }
        }
```

---
In reply to: [568229445](https://github.com/Azure/sonic-buildimage/pull/6633#discussion_r568229445) [](ancestors = 568229445)

Reply:
Got it now. Done